iclr_wrap_up/hyperparameter.json

```diff
@@ -0,0 +1,46 @@
+{ "hyperparameter": [
```

Comment:
does the "hyperparameter" level add any information? is there anything else in the hyperparameter.json?